src/app/contexts/layout.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import buildContext from '~/components/jsx-helpers/build-context';
+import loadable from 'react-loadable';
+import LoadingPlaceholder from '~/components/loading-placeholder/loading-placeholder';
+
+// Webpack wasn't able to make the dynamic strings work in the Context value.
```

Comment:
sometimes webpack does an unfortunate thing where if you use a variable in an import it loads the entire directory, this is happening in rex with the intl polyfil and its horrible